### PR TITLE
Allow empty list as input to Compose.

### PR DESCRIPTION
**Pull request by efirdc**
**Description**

This PR is just a small suggestion to allow empty lists as an input to a transform `Compose`. At the moment an error is raised in this case, but I think it would be nicer to return a null transform. This would be important if you are building a list of transforms dynamically and there is a chance of having an empty list. If the empty list case is not explicitly handled then this will introduce a hidden bug.

I ran into this situation when I was trying to construct a transform inverse, but only including the label transformations:

```python
label_transformations = []
for transform in subject.history:
    if type(transform) in (tio.RemapLabels, tio.RemoveLabels, tio.SequentialLabels):
        label_transformations.append(transform)
inverse_label_transform = tio.Compose(label_transformations ).inverse(warn=False)
```
This code worked until a case where the subject did not have any label transformations applied.

**Checklist**

- [x] I have read the [`CONTRIBUTING`](https://github.com/fepegar/torchio/blob/master/CONTRIBUTING.rst) docs and have a developer setup (especially important are `pre-commit`and `pytest`)
- [x] Non-breaking change (would not break existing functionality)
- [ ] Breaking change (would cause existing functionality to change)
- [x] Tests added or modified to cover the changes
- [x] Integration tests passed locally by running `pytest`
- [ ] In-line docstrings updated
- [ ] Documentation updated, tested running `make html` inside the `docs/` folder
- [x] This pull request is ready to be reviewed
- [x] If the PR is ready and there are multiple commits, I have [squashed them and force-pushed](https://www.w3docs.com/snippets/git/how-to-combine-multiple-commits-into-one-with-3-steps.html#force-pushing-commits-7)

### CODE REVIEW

File: torchio/transforms/augmentation/composition.py [hunk 1]
```diff
@@ -65,12 +63,9 @@ def inverse(self, warn: bool = True) -> Transform:
...
                 warnings.warn(message, RuntimeWarning)
         transforms.reverse()
-        if transforms:
-            result = Compose(transforms)
-        else:  # return noop if no invertible transforms are found
-            def result(x): return x  # noqa: E704
-            if warn:
-                warnings.warn('No invertible transforms found', RuntimeWarning)
+        result = Compose(transforms)
+        if not transforms:
```
Comment (fepegar) [inline torchio/transforms/augmentation/composition.py:67]: What about `if not transforms and warn`?

**Review by fepegar — CHANGES_REQUESTED** [review]
LGTM. Just a single comment in the code.

File: torchio/transforms/augmentation/composition.py (hunk 1 above)
Comment (efirdc) [reply torchio/transforms/augmentation/composition.py]: Whoops! Good catch.

Just pushed a fix.

**Comment by fepegar** [comment]
Thanks, @efirdc. Added in `v0.18.14`.

**Pull request merged**